src/dynamic_boundary_conditions/main_rainfall.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""
+@Script name: main.py
+@Description:
+@Author: pkh35
+@Date: 17/01/2022
+@Last modified by: sli229
+@Last modified date: 8/12/2022
+"""
+
+import geopandas as gpd
+import pathlib
+from shapely.geometry import Polygon
+from src.digitaltwin import setup_environment
+from src.dynamic_boundary_conditions import rainfall_sites, thiessen_polygons, hyetograph, model_input
+from src.dynamic_boundary_conditions import hirds_rainfall_data_to_db, hirds_rainfall_data_from_db
+
+
+def catchment_area_geometry_info(catchment_file) -> Polygon:
+    """
+    Extract shapely geometry polygon from the catchment file.
+
+    Parameters
+    ----------
+    catchment_file
+        The file path of the catchment polygon shapefile.
+    """
+    catchment = gpd.read_file(catchment_file)
+    catchment = catchment.to_crs(4326)
+    catchment_polygon = catchment["geometry"][0]
+    return catchment_polygon
```

Comment:
I think it would probably be better return this as gpd.GeoDataFrame here and change to Polygon only when being used. But I am open to discussion next week

Reply:
As discussed, we are leaving this as it is :)